XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -32,16 +32,47 @@
 
         <DataTemplate x:Key="NormalItemTemplate" x:DataType="x:Int32">
             <Button HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Background="{ThemeResource SystemChromeLowColor}">
-                <TextBlock Text="{x:Bind}" />
+                <TextBlock Text="{x:Bind}"/>
             </Button>
         </DataTemplate>
 
         <DataTemplate x:Key="AccentItemTemplate" x:DataType="x:Int32">
             <Button HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Background="{ThemeResource SystemAccentColor}">
-                <TextBlock Text="{x:Bind}" />
+                <TextBlock Text="{x:Bind}"/>
             </Button>
         </DataTemplate>
 
+        <DataTemplate x:Key="StringDataTemplate" x:DataType="x:String">
+            <Grid Background="{ThemeResource SystemControlBackgroundAccentBrush}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="StringDataTemplateColor" x:DataType="x:String">
+            <Grid Background="{ThemeResource SystemControlBackgroundAccentBrush}" Margin="10, 0, 10, 10">
+                <TextBlock Padding="8" Text="{x:Bind}" Foreground="{ThemeResource SystemControlForegroundChromeWhiteBrush}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            </Grid>
+        </DataTemplate>
```

Comment:
These are **so** similar, feels kind of redundant to have both. I guess the Grid's top margin the only real differentiation.